netsuite/client.bal

```diff
@@ -40,16 +40,40 @@ public isolated client class Client {
         self.basicClient = check new (config.baseURL + NETSUITE_ENDPOINT, httpClientConfig);
     }
 
+    # Creates a record instance in NetSuite according to the given detail
+    #
+    # + vendor - Details of NetSuite record instance creation
+    # + return - RecordAddResponse type record or else the relevant error 
+    @display{label: "Add New Vendor"} 
+    isolated remote function addNewVendor(@display{label: "Vendor"} NewVendor vendor) returns @display{label: "Response"} 
+                                          RecordAddResponse|error {
+        xml payload = check getPayloadAddOp(vendor, VENDOR, self.config);
+        http:Response response = check sendRequest(self.basicClient, ADD_SOAP_ACTION, payload);
+        return getResponseCreationOp(response);
+    }
+
+    # Creates a record instance in NetSuite according to the given detail
+    #
+    # + vendorBill - Details of NetSuite record instance creation
+    # + return - RecordAddResponse type record or else the relevant error
+    @display{label: "Add New Vendor Bill"}
+    isolated remote function addNewVendorBill(@display{label: "VendorBill"} NewVendorBill vendorBill) returns 
+                                              @display{label: "Response"} RecordAddResponse|error {
+        xml payload =  check getPayloadAddOp(vendorBill, VENDOR_BILL, self.config);
+        http:Response response = check sendRequest(self.basicClient, ADD_SOAP_ACTION, payload);
+        return getResponseCreationOp(response);
+    }
+
```

Comment:
Noticed that the same doc comment
`Creates a record instance in NetSuite according to the given detail`

is used for most of the remote functions.

Shall we add proper doc comments for remote operations as they will be visible to the choreo users?